### DISCUSSION TITLE
TypeScript definitions for (parsed) JSON layout output format

### DISCUSSION THREAD

**Discussion by mallison** [hpcc-systems/hpcc-js-wasm#119] [General]
Hi, my question is not directly related to this library so apologies if this is the wrong place to ask.

Do you know if there's a TypeScript definition for the result of parsing the `json` output of the `dot` command?

I've googled around and had a look on npm but can't find anything.

**Comment by GordonSmith** [comment]
There won't be any official definitions I am afraid, but what I have done in the past (for similar situations), is create a big representative output and run it through:  http://json2ts.com/ 

FWIW the official docs are here:  https://graphviz.org/docs/outputs/json/

**Reply by mallison** [reply]
Thanks for the pointer to http://json2ts.com/! I hadn't come across that before.
